### ISSUE TITLE
Fast Gradient Method Imports Missing Function optimize_linear

### ISSUE THREAD

**Issue by RylanSchaeffer** [tensorflow/cleverhans#1144]
`fast_gradient_method` has an import statement for a function that no longer exists:

`from cleverhans.future.torch.utils import optimize_linear`

https://github.com/tensorflow/cleverhans/blob/master/cleverhans/future/torch/attacks/fast_gradient_method.py#L5

It looks like the function used to exist, but no longer:

https://github.com/tensorflow/cleverhans/blob/5c1ccc259fa836d69e9582372b0d7f97140a66ef/cleverhans/future/torch/utils.py#L69

**Comment by michaelshiyu** [comment]
Hi @RylanSchaeffer , I'm on the master branch and the function seems to exist.

Which branch/fork/version of the codebase are you looking at? Could you please provide more details?

**Comment by RylanSchaeffer** [comment]
I must've misclicked on something earlier. I don't know why I thought the function isn't there. My apologies.

**Comment by michaelshiyu** [comment]
@RylanSchaeffer no worries. It happens :)
